src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
```

Comment:
It's recommended to use soft tabs set to 2 spaces for indentation, according to the task description. 🛠️